### DISCUSSION TITLE
What Node version I should use to run the app locally

### DISCUSSION THREAD

**Discussion by mahesh-gunjal** [alibaba-fusion/next#4653] [General]
I have tried several node versions but none of them worked for me.
I am getting confusing errors:
@alifd/sass-mapper@2.0.2: The engine "node" is incompatible with this module. Expected version "8". Got "18.17.1"
and If I use version 8 still getting new error:
node-sass@6.0.1: The engine "node" is incompatible with this module. Expected version ">=12". Got "8.17.0"
can anyone help me with this ?

**Comment by eternalsky** [comment]
node16 is recommend
